libs/eo/landing-page/shell/src/lib/header/eo-landing-page-shell-header.component.ts

```diff
@@ -0,0 +1,78 @@
+/**
+ * @license
+ * Copyright 2021 Energinet DataHub A/S
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License2");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import {
+  ChangeDetectionStrategy,
+  Component,
+  NgModule,
+  ViewEncapsulation,
+} from '@angular/core';
+import { RouterModule } from '@angular/router';
+import { MatButtonModule } from '@angular/material/button';
+import { MatToolbarModule } from '@angular/material/toolbar';
+import { WattButtonVariant } from '@energinet-datahub/watt';
+
+const selector = 'eo-landingpage-shell-header';
```

Comment:
suggestion: Use same naming style as the library name (kebab case)
```suggestion
const selector = 'eo-landing-page-shell-header';
```

Reply:
I ended up with: eo-landing-page-header